tools/distributed/pytorch_dist_exec.py

```diff
@@ -0,0 +1,109 @@
+import subprocess
+import os
+import socket
+from argparse import ArgumentParser, REMAINDER
+
+import torch
+
+def parse_args():
+    """
+    Helper function parsing the command line options
+    @retval ArgumentParser
+    """
+    parser = ArgumentParser(description="PyTorch Exec is a helper utiliy that "
```

Comment:
Please change the help to be a multiline string (use `"""`)

Reply:
ok